pkg/client/scout/reporter_test.go

```diff
@@ -256,9 +257,17 @@ func TestInstallID(t *testing.T) {
 		}
 	}
 	origEnv := os.Environ()
+	version.Version = "testing"
+	defer func() { version.Version = "" }()
```

Comment:
If the version changes, I think it should be restored to the value it had before the change, not simply blanked out.

Reply:
The version was already an empty string which is why I set it back to an empty string.

Reply:
Version initialization should be improved for sure. I'd like to use an embedded file instead of `--ldflags` or an `init()`, now when that's an option. In any case, I think tests that need to modify that version should make no assumptions at all, e.g.:
```
	sv := version.Version
	version.Version = "v0.0.0-bogus"
	defer func() { version.Version = sv }()
```